src/Compilers/CSharp/Test/Semantic/Semantics/OperatorTests.cs

```diff
@@ -424,7 +467,123 @@ static void Main()
+          ILocalReferenceExpression: one (OperationKind.LocalReferenceExpression, Type: S)
+      IBlockStatement (3 statements) (OperationKind.BlockStatement)
+        IExpressionStatement (OperationKind.ExpressionStatement)
+          IInvocationExpression (static void System.Console.Write(char)) (OperationKind.InvocationExpression, Type: void)
+            IArgument (Matching Parameter: value (Type: char)) (OperationKind.Argument)
+              ILiteralExpression (Text: 'l') (OperationKind.LiteralExpression, Type: char, Constant: l)
+        IIfStatement (OperationKind.IfStatement)
+          Condition: IUnaryOperatorExpression (UnaryOperationKind.BooleanLogicalNot) (OperationKind.UnaryOperatorExpression, Type: bool)
+              ILocalReferenceExpression: first (OperationKind.LocalReferenceExpression, Type: bool)
+          IBranchStatement (BranchKind.Break) (OperationKind.BranchStatement)
+        IExpressionStatement (OperationKind.ExpressionStatement)
+          IAssignmentExpression (OperationKind.AssignmentExpression, Type: bool)
+            Left: ILocalReferenceExpression: first (OperationKind.LocalReferenceExpression, Type: bool)
+            Right: ILiteralExpression (Text: false) (OperationKind.LiteralExpression, Type: bool, Constant: False)
+    IForLoopStatement (LoopKind.For) (OperationKind.LoopStatement)
+      Condition: IUnaryOperatorExpression (UnaryOperationKind.OperatorMethodTrue) (OperatorMethod: S.operator true(S) -> (Type: bool)) (OperationKind.UnaryOperatorExpression, Type: bool)
+          ILocalReferenceExpression: zero (OperationKind.LocalReferenceExpression, Type: S)
+      IBlockStatement (1 statements) (OperationKind.BlockStatement)
+        IExpressionStatement (OperationKind.ExpressionStatement)
+          IInvocationExpression (static void System.Console.Write(char)) (OperationKind.InvocationExpression, Type: void)
+            IArgument (Matching Parameter: value (Type: char)) (OperationKind.Argument)
+              ILiteralExpression (Text: 'm') (OperationKind.LiteralExpression, Type: char, Constant: m)
+    IForLoopStatement (LoopKind.For) (OperationKind.LoopStatement)
+      Condition: IUnaryOperatorExpression (UnaryOperationKind.OperatorMethodTrue) (OperatorMethod: S.operator true(S) -> (Type: bool)) (OperationKind.UnaryOperatorExpression, Type: bool)
+          ILocalReferenceExpression: one (OperationKind.LocalReferenceExpression, Type: S)
+      IBlockStatement (2 statements) (OperationKind.BlockStatement)
+        IExpressionStatement (OperationKind.ExpressionStatement)
+          IInvocationExpression (static void System.Console.Write(char)) (OperationKind.InvocationExpression, Type: void)
+            IArgument (Matching Parameter: value (Type: char)) (OperationKind.Argument)
+              ILiteralExpression (Text: 'n') (OperationKind.LiteralExpression, Type: char, Constant: n)
+        IBranchStatement (BranchKind.Break) (OperationKind.BranchStatement)",
+"Main");
```

Comment:
I agree - we should primarily use full operation tree verification only for small method bodies. I extended this test only to demonstrate the dump format.